sdk/core/core-http/src/proxyAgent.ts

```diff
@@ -7,24 +7,20 @@ import * as tunnel from "tunnel";
 
 import { ProxySettings } from "./serviceClient";
 import { URLBuilder } from "./url";
-import { HttpHeaders } from "./httpHeaders";
+import { HttpHeadersLike } from "./httpHeaders";
 
 export type ProxyAgent = { isHttps: boolean; agent: http.Agent | https.Agent };
 export function createProxyAgent(
   requestUrl: string,
   proxySettings: ProxySettings,
-  headers?: HttpHeaders
+  headers?: HttpHeadersLike
 ): ProxyAgent {
   const host = URLBuilder.parse(proxySettings.host).getHost() as string;
   if (!host) {
-    throw new Error(
-      "Expecting a non-empty host in proxy settings."
-    );
+    throw new Error("Expecting a non-empty host in proxy settings.");
   }
   if (!isValidPort(proxySettings.port)) {
-    throw new Error(
-      "Expecting a valid port number in the range of [0, 65535] in proxy settings."
-    );
+    throw new Error("Expecting a valid port number in the range of [0, 65535] in proxy settings.");
```

Comment:
Is this after running Prettier on everything?  Not sure if the column width might have been changed between now and the last time this was formatted.

Reply:
yeah this was an autoformat change